jenkins-design-language/src/js/components/ResultItem.jsx

```diff
@@ -78,6 +79,11 @@ export class ResultItem extends Component {
         }
     };
 
+    urlClicked = (e: Event) => {
+        console.log('urlClicked');
```

Comment:
Probably just make sure we remove that before PR merge?

Reply:
Nice catch, cheers